scripts/llnl_scripts/llnl_lc_build_tools.py

```diff
@@ -378,6 +379,34 @@ def build_and_test_host_config(test_root, host_config,
             print("[ERROR: Installed 'using-with-blt' example for host-config: %s failed]\n\n" % host_config)
             return res
 
+    if should_test_installed_tutorial:
+        # Notes: Might require loading module for more recent cmake than the system default for some platforms
+        install_example_dir = pjoin(install_dir, "examples", "axom", "radiuss_tutorial")
+        install_example_output_file = pjoin(build_dir,"output.log.install_example.radiuss_tutorial.txt")
+        print("[testing installed 'radiuss_tutorial' example]")
+        print("[log file: %s]" % install_example_output_file)
+
+        example_commands = [
+            "cd {0}".format(install_example_dir),
+            "rm -rf build",
+            "mkdir build",
+            "cd build",
+            """echo "[Configuring 'radiuss_tutorial']" """,
+            "cmake -C ../host-config.cmake -DCMAKE_BUILD_TYPE={0} ..".format(build_type),
+            """echo "[Building 'radiuss_tutorial']" """,
+            "make -j16",
```

Comment:
16 may be generous here and on the ctest command.. I would go with 8 at most.

Reply:
Sure. We use `-j16` for building in a bunch of places.
Are you suggestion -`j8` for ctest, but `-j16` for building?

Reply:
I must be mistaking this for Serac... we ran into a problem and had to lower the job parallism. You are right we have 16 all over. Its fine to just leave it then.